src/components/Header/index.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+import * as S from './styles';
+import Navbar from '../Navbar';
+
+const Header = () => {
+    return (
+        <S.DivWrapper>
```

Comment:
remove 'Div' from the name here to be consistent with the naming

Reply:
done.